shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/utility/LocationUtil.kt

```diff
@@ -7,7 +7,7 @@ typealias Degree = Double
 
 class LocationUtil {
 
-    fun getDistance(lat1: Double, lon1: Double, lat2: Double, lon2: Double, inMiles: Boolean): Double {
+    fun getDistance(lat1: Double, lon1: Double, lat2: Double, lon2: Double, distanceUnit: Int): Double {
```

Comment:
Could DistanceUnit be used as the type instead of Int, or does that cause issues with the shared code?

Reply:
I wasn't sure whether it would play nice, so was keeping it agnostic in first instance.